src/lib/routes/admin-api/user/pat.ts

```diff
@@ -48,7 +52,17 @@ export default class PatController extends Controller {
                 openApiService.validPath({
                     tags: ['Personal access tokens'],
                     operationId: 'getPats',
-                    responses: { 200: createResponseSchema('patsSchema') },
+                    summary: 'Get all current users Personal Access Tokens.',
+                    description:
+                        'Returns all of the [Personal Access Tokens](https://docs.getunleash.io/how-to/how-to-create-personal-access-tokens) belonging to the current user.',
+                    responses: {
+                        200: createResponseSchema('patsSchema'),
```

Comment:
Should we split `patsSchema` into separate request and response types?

Reply:
Yeah, probably. Because you can't provide the same data when you create one, right?

Reply:
Yeah indeed, it would be ignored if you tried no? I'll create a task for this under the technical debt tag